packages/core/bootstrap/src/lib/modules/requester.ts

```diff
@@ -355,6 +420,11 @@ interface SingleResult {
   result?: number | string
 }
 
+interface SingleResultWithMarketOpen {
```

Comment:
How common with marketOpen flags be?
I would rather make this into a generic, and then if we want to hold a re-usable interface to pass in as `additionalFields` we can put it somewhere else.